dev-docs/modules/contxtfulRtdProvider.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page_v2
+title: Contxtful RTD Provider  
+display_name: Contxtful RTD Provider  
+description: Activates Receptivity to evaluate ad success in real-time by focusing on attention and context.
+page_type: module
+module_type: rtd
+module_code : contxtfulRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# Contxtful Real Time Data Module
+
+## Overview
+
+The Contxtful RTD module enhances ad units by adding a Receptivity score to the meta-data. This score quantifies user attention in real time and can be used to guide bidding decisions.
+
+{% include dev-docs/loads-external-javascript.md %}
```

Comment:
There's a build error in the preview

> Liquid Exception: Could not locate the included file "dev-docs/loads-external-javascript.md" in any of ["/opt/build/repo/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in dev-docs/modules/contxtfulRtdProvider.md

I have no laptop at hand, but either the path is wrong or there's some hickup in the preview.

Reply:
@muuki88 - I think this is just an issue with the preview. Other RTD modules do this same thing, and it works in prod.